src/components/Navbar.css

```diff
@@ -0,0 +1,72 @@
+.navbar {
+  background-color: #060b26;
+  height: 80px;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.menu-bars {
+  margin-left: 2rem;
+  font-size: 2rem;
+  background: none;
+}
+
+.nav-menu {
+  background-color: #060b26;
```

Comment:
Just a nit, ik this is a wip but let's try to define our colour theme in a global constants file somewhere and reference that instead of raw hex literals like this.

The colours are all already defined if we do decide to use Tailwind, if not let's do it in a global css or something